syntax/supercollider.vim

```diff
@@ -39,6 +39,7 @@ syn match	scSymbol "\w\+:"
 
 syn region  scString start=+"+ skip=+\\\\\|\\"+ end=+"+
 
+syn match	scChar	"\$[^ ]"
```

Comment:
Not sure right now but with that line added we might be able to remove the 3 following below..